core/src/navigation/LogoTitle.svelte

```diff
@@ -51,7 +54,9 @@
               pathData = obj.pathData;
             } else if (prop === 'appSwitcherItems') {
               appSwitcherItems = obj.appSwitcherItems;
-            } else if (prop === 'pathParams') {
+            } else if (prop === 'customAppSwitcherItemRenderer') {
+              customAppSwitcherItemRenderer = obj.customAppSwitcherItemRenderer;
+            }else if (prop === 'pathParams') {
```

Comment:
```suggestion
            } else if (prop === 'pathParams') {
```
It seems like linting did not work properly.